API/DeleteTextBackTranslation.php

```diff
@@ -0,0 +1,35 @@
+<?php
+require_once('utils/Model.php');
+require_once('utils/Respond.php');
+
+session_start();
+
+
+if ($_SERVER['REQUEST_METHOD'] !== 'POST') {
+    RespondWithError(405, "Request must be a POST");
+}
+
+if (!(
+    array_key_exists('PhoneId', $_POST) &&
+    array_key_exists('term', $_POST))
+) {
+    RespondWithError(400, 'This endpoint requires PhoneId & term.');
+}
+
+$androidId = trim($_POST['PhoneId']);
+$term = trim($_POST['term']);
+
+$model = new Model();
+# check if project exists for the given androidId
+$projectId = $model->GetProjectId($androidId);
+
+if (!$projectId) {
+    RespondWithError(400, "Please register a project using /API/RegisterPhone.php before using /API/UploadTextBackTranslation.php");
```

Comment:
please update to "API/DeleteTextBackTranslation.php"

Reply:
Ahhh... it's copy paste. I think we should 404 if delete endpoint is called. 

Let's tell end user to create a project and then upload translations and then delete it.

I am happy with either way.